packages/@uppy/provider-views/src/ProviderView/ProviderView.jsx

```diff
@@ -1,5 +1,7 @@
 import { h } from 'preact'
 
+import { getSafeFileId } from '@uppy/core/src/Uppy.js'
```

Comment:
```suggestion
import { getSafeFileId } from '@uppy/core/lib/Uppy.js'
```

Reply:
Wow such a subtle bug. This import want generated by vscode autocompletion. I wonder if we could add some linting to prevent importing from src